.coveragerc

```diff
@@ -3,6 +3,7 @@ branch = True
 include = glotaran/*
 omit = setup.py
        */tests/*
```

Comment:
`tests` doesn't exist anymore, so we should not omit it in case we move stuff there later.

Reply:
``tests`` does exists at ``glotaran/models/spectral_temporal/tests``

Reply:
ok, i got it wrong